src/vs/workbench/contrib/debug/browser/debug.contribution.ts

```diff
@@ -213,6 +213,11 @@ configurationRegistry.registerConfiguration({
 			default: 'onFirstSessionStart'
 		},
 		'debug.internalConsoleOptions': INTERNAL_CONSOLE_OPTIONS_SCHEMA,
+		'debug.closeConsoleOnEnd': {
```

Comment:
The setting should be called `debug.console.closeOnEnd`.

Also the descriptpon needs to be updated, it is wrong